src/GitTfs/Program.cs

```diff
@@ -25,7 +25,17 @@ public static void Main(string[] args)
         {
             try
             {
+                var sw = Stopwatch.StartNew();
+
                 Environment.ExitCode = MainCore(args);
+
+                sw.Stop();
+                // could write it out like this: mm\\:ss\\.ff, using readable format instead
+                Console.WriteLine("\r\nCompleted in {0}{1}{2}{3}",
+                    FormatTime(sw.Elapsed.Days, "day"), //hopefully this should never happen??? :)
+                    FormatTime(sw.Elapsed.Hours, "hour"),
+                    FormatTime(sw.Elapsed.Minutes, "minute"),
+                    FormatTime(sw.Elapsed.Seconds, "second"));
```

Comment:
I don't think this time measurement should be merged.
Something similar can be archived by simply running git tfs with the `time` command, e.g. have a look at https://linuxize.com/post/linux-time-command for further details.

Reply:
No worries. I am using powershell to automate the pulls / fetches. I will just wrap the git-tfs command with measure-command.